contexts/webledger-v1.jsonld

```diff
@@ -71,6 +72,7 @@
     "normalizationAlgorithm": "sec:normalizationAlgorithm",
     "object": {"@id": "wl:object", "@type": "@id"},
     "operation": {"@id": "wl:operation", "@type": "@id", "@container": ["@graph", "@set"]},
+    "operationHash": {"@id": "wl:operationHash", "@container": "@set"},
```

Comment:
It seems like we may not always want this to be an `@set`. Right? If so, we'll need to make this look like the `eventHash` definition and then make sure that the code handles both cases (e.g. normalizes to an array where it makes sense to do so).

Reply:
afaik `eventHash` is only used in `meta` so maybe it doesn't need to be defined in the context at all?

And the non-set version of `operationHash` currently called `opHash` is also in meta.  

But perhaps you are referring to instances where `operationHash` has only one item in which case is could be represented as a string?

My thinking was that `operationHash` was going to be patterned after `parentHash`.  In the case of regular events, `parentHash` is always an array of one item represented as an array.